OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -75,7 +75,7 @@ void IRenderAboveWorld.RenderAboveWorld(Actor self, WorldRenderer wr)
 				s.Info.Type.DrawCombatOverlay(wr, wcr, self);
 
 			var tc = Color.Lime;
-			var positions = activeShapes.SelectMany(tp => tp.TargetablePositions(self));
+			var positions = Target.FromActor(self).Positions;
```

Comment:
Can you elaborate on this change?

Reply:
Making the interface explicit also made `TargetablePositions` non-public, and I felt this approach is simpler than adding more stuff to `HitShape` just for the CombatOverlay.

Reply:
Ok, makes sense.  In practice this shouldn't change what is rendered, right?

Reply:
It shouldn't, and I didn't notice any issues while testing this.